packages/swapper/src/swappers/RelaySwapper/constant.ts

```diff
@@ -0,0 +1,48 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import {
+  arbitrumChainId,
+  avalancheChainId,
+  baseChainId,
+  btcChainId,
+  CHAIN_REFERENCE,
+  ethChainId,
+  gnosisChainId,
+  optimismChainId,
+  polygonChainId,
+  solanaChainId,
+} from '@shapeshiftoss/caip'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import { arbitrum, avalanche, base, gnosis, optimism, polygon } from 'viem/chains'
+
+import type { SupportedChainIds } from '../../types'
+
+export const relayChainMap: Record<ChainId, number> = {
+  [btcChainId]: 8253038,
+  [ethChainId]: Number(CHAIN_REFERENCE.EthereumMainnet),
+  [arbitrumChainId]: arbitrum.id,
+  [baseChainId]: base.id,
+  [optimismChainId]: optimism.id,
+  [polygonChainId]: polygon.id,
+  [solanaChainId]: 792703809,
+  [gnosisChainId]: gnosis.id,
+  [avalancheChainId]: avalanche.id,
+}
+
+export const DEFAULT_RELAY_EVM_TOKEN_ADDRESS = '0x0000000000000000000000000000000000000000'
```

Comment:
nitpick: viem `zeroAddress`, perhaps as an alias if we want to keep declarative

Reply:
Done in [7e3a115](https://github.com/shapeshift/web/pull/9188/commits/7e3a115c7aeb37843ca13844d6f4f73501a40f03)